src/lib/types/model.ts

```diff
@@ -365,3 +365,66 @@ export interface IFeatureStrategySegment {
     featureStrategyId: string;
     segmentId: number;
 }
+
+export interface ISuggestChangeset {
+    id: number;
+    state: string;
+    project: string;
+    environment: string;
+    createdBy?: Pick<User, 'username' | 'imageUrl'>;
+    createdAt?: Date;
+    changes?: ISuggestChange[];
```

Comment:
Can you have a change set without changes? Or a change set with an empty list of changes for that matter? If not, maybe it makes sense to make it a required, non-empty list? That is, something like `changes: [ISuggestChange, ...ISuggestChange[]]`?